node/heartbeat/monitor_test.go

```diff
@@ -99,11 +99,13 @@ func TestNewMonitor_ShouldComputeShardId(t *testing.T) {
 		0: {"pk0"},
 		1: {"pk1"},
 	}
+
+	maxDuration := 1 * time.Millisecond
```

Comment:
maybe remove 1 *

Reply:
I think it is more readable this way.

Reply:
no need for the 1 *. it is remove from the whole project.

Reply:
ok, removed